src/Integration/Symfony/Form/WebFileType.php

```diff
@@ -89,11 +103,19 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'error_mapping' => []
             ]);
 
-            $builder->add(self::FILE_FIELD, WebFileType::class, $fileFieldOptions);
+            $builder->add(self::FILE_FIELD, self::class, $fileFieldOptions);
             $builder->add(self::REMOVE_FIELD, CheckboxType::class, $removeFieldOptions);
 
             $builder->addEventListener(FormEvents::PRE_SUBMIT, new RemovableWebFileListener());
             $builder->addModelTransformer(new RemovableFileTransformer(self::FILE_FIELD));
+        } elseif (true === $options['multiple']) {
+            if (null === $this->multipleFileTransformers) {
```

Comment:
You could simply have made a `getMultipleFileTransformers` method that would initialize them if there are not already.